crates/pop-cli/src/commands/bench/machine.rs

```diff
@@ -75,7 +75,8 @@ impl BenchmarkMachine {
 		let mut arguments: Vec<String> = std::env::args().skip(3).collect();
 		#[cfg(test)]
 		{
-			arguments.retain(|arg| arg != "--show-output" && arg != "--nocapture");
+			arguments
+				.retain(|arg| arg != "--show-output" && arg != "--nocapture" && arg != "--ignored");
```

Comment:
I think this is cleaner:


```suggestion
				.retain(|arg| !matches!(arg.as_str(), "--show-output" | "--nocapture" | "--ignored"));
```

Reply:
I like your suggestion, refactored in https://github.com/r0gue-io/pop-cli/pull/649/commits/2a56f2ac9abec34ebf478efa67d2bd6b4359eed5